cotk/metric/bleu.py

```diff
@@ -204,17 +204,28 @@ def close(self):
 		_ref = _replace_unk(ref, self.dataloader.unk_id)
 
 		bleu_irl = []
-		if self.sample >= 1000 and self.cpu_count > 1:
+
+		if self.sample > 1:

```

Comment:
add at line 198 ``if self.sample <= 1``  should raise Error.   remove ``if`` here.